### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by bluquar**
1. Turned on bracket spacing to conform to the style I've seen in open source code so far and to improve readability
2. Reorganized directories to have "src/gp" (general purpose) and "src/ca" (community aid)
3. Split out Themed.tsx into individual View and Text modules
4. Switched to using absolute imports

### CODE REVIEW

**Comment by lowellbander** [comment]
what open source code have u been reading?

why acronym directory names rather than "longform"?

File: src/gp/components/TimeAgo.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import TimeAgo from 'javascript-time-ago';
+import JavascriptTimeAgo from 'javascript-time-ago';
```
Comment (lowellbander) [inline src/gp/components/TimeAgo.tsx:1]: is this a synonym for TimeAgo?

File: src/gp/components/light_or_dark_themed/useThemeColor.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import useColorScheme from 'gp/hooks/useColorScheme';
+
+export function useThemeColor(
+  props: { light?: string; dark?: string },
+  colorName: keyof typeof Colors.light & keyof typeof Colors.dark,
+) {
+  const theme = useColorScheme();
+  const colorFromProps = props[theme];
+
+  if (colorFromProps) {
```
Comment (lowellbander) [inline src/gp/components/light_or_dark_themed/useThemeColor.ts:16]: isn't this just `return colorFromProps ?? Colors[theme][colorName]`? is that not allowed in typescript?

File: src/gp/hooks/useCachedResources.ts [hunk 3]
```diff
@@ -16,7 +16,7 @@ export default function useCachedResources() {
         await Promise.all([
           Font.loadAsync({
             ...FontAwesome.font,
-            'space-mono': require('../assets/fonts/SpaceMono-Regular.ttf'),
+            'space-mono': require('../../../assets/fonts/SpaceMono-Regular.ttf'),
```
Comment (lowellbander) [inline src/gp/hooks/useCachedResources.ts:19]: lol

**Review by lowellbander — APPROVED** [review]

**Comment by bluquar** [comment]
> what open source code have u been reading?
> 
> 
> 
> why acronym directory names rather than "longform"?

Tutorials and documentation for the open source libraries I'm using!

Acronyms just to shorten up the import lines. I'm open to more descriptive directory names tho

**Comment by lowellbander** [comment]
yay copy on the libs! i figure descriptive is usually best, so like a
director called common/ is a lot more clear than gp/

File: src/gp/components/TimeAgo.tsx (hunk 1 above)
Comment (bluquar) [reply src/gp/components/TimeAgo.tsx:1]: I don't understand the question. TimeAgo now refers to the component exported by this module

File: src/gp/components/light_or_dark_themed/useThemeColor.ts (hunk 2 above)
Comment (bluquar) [reply src/gp/components/light_or_dark_themed/useThemeColor.ts:16]: I believe that is allowed. this is copied from Themed.tsx which is one of the files that was generated when I created the expo project. that's why the style is a bit different than what you'd probably see me write :)

File: src/gp/hooks/useCachedResources.ts (hunk 3 above)
Comment (bluquar) [reply src/gp/hooks/useCachedResources.ts:19]: hehe

**Comment by bluquar** [comment]
> yay copy on the libs! i figure descriptive is usually best, so like a director called common/ is a lot more clear than gp/

#55

**Pull request merged**